modules/ROOT/pages/servicenow/servicenow-connector-studio.adoc

```diff
@@ -35,21 +36,81 @@ Anypoint Studio provides two ways to add the connector to your Studio project: f
 
 == Configure the Connector
 
-Refer to these values when configuring your connector.
+The ServiceNow Connector supports Basic and OAuth2 authentications. Select the authorization type to use from the *Connection* dropdown.
+
+image::servicenow/servicenow-studio-connection-dropdown.png[]
+
+
+=== Basic Authentication
+
+. In the ServiceNow Config *General* tab, select *Basic Authentication* from the *Connection* drop-down list. 
+. Provide the following values for Basic authentication:
 
 [%header%autowidth.spread]
 |===
 |Field |Description
-|*Username* | Username for log in
-|*Password* | Corresponding password
+|*Username* | User name for logging in to ServiceNow
+|*Password* | Password that corresponds to the username
 |*Service Address* | The WSDL endpoint
-|*ServiceNow Version* | The ServiceNow version being used
-|*User table list*| Custom tables for a user that is logged in
+|*ServiceNow Version* | The ServiceNow version to use
+|*User table list*| Custom tables for a user who is logged in
+|===
+
+==== Basic Auth HTTP Message Dispatcher Provider
+
+. Click the *Transport* tab. 
+. From *Transport Configuration*, select *Basic auth http message dispatcher provider*.
+. Enter the following required values: 
+
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*User Name* | User name for logging in to ServiceNow
+|*Password* | Password that corresponds to the user name
+|===
+  
+==== Basic Auth HTTP Requester Based Transport Provider
+
+. Click the *Transport* tab. 
+. From *Transport Configuration*, select *Http requester based transport provider*.
+. Select the *Requester config* or click the green plus sign to create a new one. 
++
+image::servicenow/servicenow-studio-requester-config.png[]
++
+. Specify the required values:
+
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*Host* | The WSDL endpoint
+|*Username* | User name for logging in to ServiceNow
+|*Password* | Password that corresponds to the user name
+|===
+
+=== OAuth2.0 Authentication
+
+. In the ServiceNow Config *General* tab, select *OAuth 2.0 Authorization Code* from the *Connection* drop-down list. 
+. Specify the values for the OAuth connection: 
+
+image::servicenow/servicenow-oauth.png[]
+
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*Service Address* | ServiceNow's instance endpoint
+|*Consumer key* | Client ID from the registered application
+|*Consumer secret* | Client secret from the registered application
+|*Authorization url* | Endpoint for initiating the OAuth dance
+|*Access token url* | Endpoint for retrieving the access token
+|*Listener config* | HTTP Listener configuration
+|*Callback path* | The path of the access token callback endpoint
+|*Authorize path* | The path of the local HTTP endpoint that triggers OAuth dance
+|*External callback url* | If the callback endpoint is behind a proxy  or should be accessed through a non-direct URL, use this parameter specify the URL the OAuth provider should use to access the callback.
```

Comment:
use this parameter specify the URL > use this parameter **_to_** specify the URL